CHANGELOG.md

```diff
@@ -1,4 +1,14 @@
 ## 3.72.0 (Unreleased)
+IMPROVEMENTS:
+* compute: added support for IPsec-encrypted Interconnect in the form of new fields on `google_compute_router`, `google_compute_ha_vpn_gateway`, `google_compute_interconnect_attachment` and `google_compute_address`([#9288](https://github.com/hashicorp/terraform-provider-google/pull/9288))
+* container: Allowed specifying a cluster id field for `google_container_node_pool.cluster` to ensure that a node pool is recreated if the associated cluster is recreated. ([#9309](https://github.com/hashicorp/terraform-provider-google/pull/9309))
+* storage-transfer: added support for `azure_blob_storage_data_source` to `google_storage_transfer_job` ([#9311](https://github.com/hashicorp/terraform-provider-google/pull/9311))
```

Comment:
should this just be `storage` or is `storage-transfer` separate?

Reply:
Looks like `google_storage_transfer_job` is under its own tab in the docs: https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/storage_transfer_job

So storage-transfer is probably correct

Reply:
changed to `storagetransfer`